Source/WebCore/accessibility/AccessibilityTableCell.cpp

```diff
@@ -152,20 +152,20 @@ AccessibilityRole AccessibilityTableCell::determineAccessibilityRole()
     
 bool AccessibilityTableCell::isTableHeaderCell() const
 {
-    auto* node = this->node();
-    if (!node)
+    auto* element = this->element();
+    if (!element)
         return false;
 
-    if (node->hasTagName(thTag))
+    if (element->elementName() == ElementName::HTML_th)
```

Comment:
Can we get the elementName() once instead of doing it multiple times? That would be more succinct, and more performant.